Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -189,6 +189,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
     self.viewModel.outputs.metadataData
       .observeForUI()
       .observeValues { [weak self] data in
+        self?.metadataIconImageView.isHidden = data.iconImage == nil
```

Comment:
@theginbin any idea why so many this was originally done with so many side-effects in a single `observeValues`?. I think if we did this today we'd have had an output for each of those and been more explicit about the logic in the view model.